### PR TITLE
Another liberty port for trilby

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
Ports: https://github.com/Liberty-Landing/Liberty-Station-13/pull/358 at Trilbys request.

Adds a new perk, available through background or tasks that allows you to nullify one of your existing perks. 

Edit; also fixes a teeny tiny bug with some other code that was causing linters fails. 
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/datums/perks/cooldown.dm [hunk 1]
```diff
@@ -49,3 +49,101 @@
 	if(holder)
 		holder.stats.changeStat(STAT_COG, 5)
 	..()
+<<<<<<< HEAD
```
Comment (cdb-is-not-good) [inline code/datums/perks/cooldown.dm:52]:
```suggestion

```

**Comment by cdb-is-not-good** [comment]
Alright there we go, bit messy with the commits but hey.

File: code/datums/perks/special.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/////////////////
+//Task Perks//
+/////////////////
+
+/datum/perk/forceful_rejection
+	name = "Forecful Rejection"
+	desc = "At the cost of a lot of your mental health, and willpower, remove a perk of your choice!"
+	gain_text = "This reality is unfair and cruel! I should just forget about it..."
+	lose_text = "It takes a lot of effert to reject reality."
```
Comment (SingingSpock) [inline code/datums/perks/special.dm:9]: *effort

Comment (cdb-is-not-good) [inline code/datums/perks/special.dm:9]:
```suggestion
	lose_text = "It takes a lot of effort to reject reality."
```

File: code/datums/perks/special.dm [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	if(!istype(user))
+		return ..()
+	if(anti_cheat)
+		to_chat(usr, SPAN_NOTICE("You cant reject well rejecting!"))
+		return FALSE
+
+	anti_cheat = TRUE
+
+	if(user.stats.perks.len ==0)
+		to_chat(user, "Creature has no perks to remove, not sure how thats possable")
```
Comment (cdb-is-not-good) [inline code/datums/perks/special.dm:36]:
```suggestion
		to_chat(user, "Creature has no perks to remove, not sure how that's possible.")
```

File: code/datums/perks/special.dm [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+		var/mob/living/carbon/human/user = usr
+		user.sanity.change_max_level(-10)
+	..()
+
+/datum/perk/forceful_rejection/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(anti_cheat)
+		to_chat(usr, SPAN_NOTICE("You cant reject well rejecting!"))
```
Comment (cdb-is-not-good) [inline code/datums/perks/special.dm:30]:
```suggestion
		to_chat(usr, SPAN_NOTICE("You cant reject while rejecting!"))
```

File: code/datums/perks/special.dm [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+		anti_cheat = FALSE
+		to_chat(user, SPAN_NOTICE("No need to reject some of my past."))
+		return
+	if(QDELETED(user))
+		to_chat(user, "You cant reject being removed from this plane of reality...")
+		return
+	anti_cheat = FALSE
+	user.stats.removePerk(perkname.type)
+	user.stats.removePerk(PERK_FORCEFUL_REJECTION)
+	message_admins("\blue [key_name(user)] removed the perk [perkname] from using forceful rejection.", 1)
```
Comment (cdb-is-not-good) [inline code/datums/perks/special.dm:50]:
```suggestion
	message_admins("\blue [key_name(user)] removed the perk [perkname] by using forceful rejection.", 1)
```

File: code/datums/perks/special.dm [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+	icon_state = "reject_reality"
+	active = FALSE
+	passivePerk = FALSE
+	var/anti_cheat= FALSE
+
+/datum/perk/forceful_rejection/assign(mob/living/carbon/human/H)
+	..()
+
+/datum/perk/forceful_rejection/remove()
+	holder.stats.changeStat(STAT_VIV, -15)
```
Comment (Trilbyspaceclone) [inline code/datums/perks/special.dm:19]:
```suggestion
	holder.stats.changeStat(STAT_VIV, -15)
	holder.calculate_nsa()
```

**Pull request merged**